pygeos/test/test_set_operations.py

```diff
@@ -168,3 +292,43 @@ def test_coverage_union_non_polygon_inputs(geom_1, geom_2):
         pygeos.GEOSException, match="Unhandled geometry type in CoverageUnion."
     ):
         pygeos.coverage_union(geom_1, geom_2)
+
+
+@pytest.mark.skipif(pygeos.geos_version < (3, 9, 0), reason="GEOS < 3.9")
+@pytest.mark.parametrize(
+    "geom,grid_size,expected",
+    [
+        # floating point precision, expect no change
+        (
+            [pygeos.box(0.1, 0.1, 5, 5), pygeos.box(0, 0.2, 5.1, 10)],
+            0,
+            pygeos.Geometry(
+                "POLYGON ((0 0.2, 0 10, 5.1 10, 5.1 0.2, 5 0.2, 5 0.1, 0.1 0.1, 0.1 0.2, 0 0.2))"
+            ),
+        ),
+        # grid_size is at effective precision, expect no change
+        (
+            [pygeos.box(0.1, 0.1, 5, 5), pygeos.box(0, 0.2, 5.1, 10)],
+            0,
```

Comment:
```suggestion
            0.1,
```

? 

(otherwise it is the same test as the one above)